conf/cassandra.yaml

```diff
@@ -241,10 +241,10 @@ credentials_validity_in_ms: 2000
 # credentials_update_interval_in_ms: 2000
 
 # If true, cache contents are actively updated by a background task at the
-# interval set by credentials_update_interval_in_ms. If false, cache entries
+# interval set by credentials_update_interval_in_ms. If false (default), cache entries
 # become eligible for refresh after their update interval. Upon next access,
 # an async reload is scheduled and the old value returned until it completes.
-# credentials_cache_active_update: true
+# credentials_cache_active_update: false
```

Comment:
Should we do this for `permissions_cache_active_update` and `roles_cache_active_update` as well?

Reply:
👍 : Will do that on commit. Running CI right now.